routingv8/request.go

```diff
@@ -70,7 +70,9 @@ const (
 
 type GeoWaypoint struct {
 	// Elevation is available in responses when using the ElevationReturnAttribute
-	Elevation float64 `json:"elv"`
+	// TODO(odsod): Consider refactoring this to another message.
```

Comment:
Does your comment mean that it's kinda like a "WRITE_ONLY" field? I.e. the Elevation field must be omitted in all responses, but is allowed to be set in all requests?

Reply:
Not sure if it applies to all requests, but there are at least some where it triggers an error